src/Symfony/Component/Amqp/Command/AmqpMoveCommand.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Amqp\Command;
+
+use Psr\Container\ContainerInterface;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Amqp\Broker;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+class AmqpMoveCommand extends Command
+{
+    private $container;
+    private $logger;
+
+    /**
+     * @param ContainerInterface   $container A PSR11 container from which to load the Broker service
+     * @param LoggerInterface|null $logger
+     */
+    public function __construct(ContainerInterface $container, LoggerInterface $logger = null)
+    {
+        parent::__construct();
+
+        $this->container = $container;
+        $this->logger = $logger;
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('amqp:move')
```

Comment:
to make the command lazy, you should define `protected static $defaultName = 'amqp:move';` instead